tests/e2e/specs/upe-split/shopper/shopper-upe-enabled-all-flows.spec.js

```diff
@@ -11,16 +11,20 @@ import { fillCardDetails, setupProductCheckout } from '../../../utils/payments';
 const { shopper, merchant, uiUnblocked } = require( '@woocommerce/e2e-utils' );
 
 const MIN_WAIT_TIME_BETWEEN_PAYMENT_METHODS = 20000;
-
-const sepaPaymentMethod = '#inspector-checkbox-control-8';
+const upeMethodCheckboxes = [
```

Comment:
```suggestion
const UPE_METHOD_CHECKBOXES = [
```

Reply:
Good shout to correct the convention here: amended in 017e819c94a77dcbead5c94b7f758eb63e8e353b.